dev/AppLifecycle/Association.cpp

```diff
@@ -67,9 +67,16 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 
         std::hash<std::wstring> hasher;
         auto hash = hasher(seed);
-;
+        uint64_t hash64 = static_cast<uint64_t>(hash);
```

Comment:
NIT: `uint64_t hash64{...};'  --> [Prefer {}=initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

NIT: `auto hash64{...};` --> auto instead of redundantly specifying type

NIT: `uint64_t hash64{ hash };` --> even better, drop the cast and let the compiler do it via the initializer syntax